lib/full_text_search/issue_content_synchronizable.rb

```diff
@@ -0,0 +1,38 @@
+module FullTextSearch
+  module IssueContentSynchronizable
+    extend ActiveSupport::Concern
+
+    included do
+      after_commit :queue_sync_on_commit
+      after_destroy :queue_sync_on_destroy
+    end
+
+    private
+
+    def queue_sync(action, options = {})
+      FullTextSearch::UpdateIssueContentJob.perform_later(
+        self.class.name,
+        self.id,
+        action,
+        options
+      )
+    end
+
+    def queue_sync_on_commit
+      queue_sync("commit")
+    end
+
+    def queue_sync_on_destroy
+      case self
+      when Issue
+        queue_sync("destroy")
+      when Journal
+        queue_sync("destroy", issue_id: self.journalized_id)
+      when Attachment
+        if self.container_type == Issue.name
+          queue_sync("destroy", issue_id: self.container_id)
+        end
+      end
```

Comment:
Can we use separated module instead of dispatching in a method like we use separated `Mapper` for each object?

Reply:
fix: [e790837](https://github.com/clear-code/redmine_full_text_search/pull/167/commits/e790837a63be9cfb2ee3650151219463ca6483d1) Sure, I separated this module. I simplicity named each module to update searched target content. But at  the same time, I'm thinking whether I should name it including specific column names like content, notes , or description.